test/support/minitest/assert_wkt_similar.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Support
+  module Minitest
+    module AssertWktSimilar
+      def assert_wkt_similar(a, b)
+        assert_equal(Inner.normalize(a), Inner.normalize(b), "Normalized WKT differs")
+      end
+
+      module Inner
+        module_function
+
+        def normalize(wkt)
+          generator.generate(parser.parse(wkt))
+        end
+
+        def parser
+          @parser ||= RGeo::WKRep::WKTParser.new
+        end
+
+        def generator
+          @generator ||= RGeo::WKRep::WKTGenerator.new
+        end
+      end
```

Comment:
I guess the only issue with this is that it relies on another part of the package being implemented correctly to pass our tests. I think this is a smart way to handle it and is almost definitely fine since the `WKTParser` and `WKTGenerator` classes seldom change, but could potentially cause tests to seemingly randomly fail later if we were to introduce a bug in either of these.

Reply:
@keithdoggett I thought about that as well, my excuse is that this should not be buggy, and should also be tested anyway, so if tests are failing for that reason, we can be glad they fail and take this opportunity to consolidate.

That's just an excuse though, of course I'd rather have tests less intricated.. I'll look for how the wkt readers and writers are made in GEOS, and options we could use to have something more similar

Reply:
There are definitely some options: https://libgeos.org/doxygen/geos__c_8h.html#a84d8fd4c78746d91fe0d6cffe3287ea8 (See GEOSWKTWriter_ methods for instance)

I don't know that we want to go much further down that road for now. Maybe we could find some kind of harmony between ruby and geos parser, and have the same interface for all ?

Reply:
Yeah I don't think it's necessary to do right now. This solution is likely going to suffice and assuming we have tested the `WKTParser` and `WKTGenerator` classes then it shouldn't be much of an issue because breaking changes to them would cause their tests to fail too.